src/main/java/org/team1540/elmo/Constants.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) FIRST and other WPILib contributors.
+
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.team1540.elmo;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+
+/**
+ * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
+ * constants. This class should not be used for any other purpose. All constants should be declared
+ * globally (i.e. public static). Do not put anything functional in this class.
+ *
+ * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * constants are needed, to reduce verbosity.
+ */
+public final class Constants {
+
+    // DRIVETRAIN CAN IDS
+    public static final int LEFT_FRONT_MOTOR_CAN_ID = 0;
+    public static final int LEFT_BACK_MOTOR_CAN_ID = 1;
+    public static final int RIGHT_FRONT_MOTOR_CAN_ID = 2;
+    public static final int RIGHT_BACK_MOTOR_CAN_ID = 3;
+
+    // SOLENOIDS
+    public static final int ARM_FORWARD_CHANNEL = 0;
+    public static final int ARM_REVERSE_CHANNEL = 1;
+    public static final PneumaticsModuleType ARM_TYPE = PneumaticsModuleType.CTREPCM;
+    public static final int SUCTION_CHANNEL = 2;
+    public static final PneumaticsModuleType SUCTION_TYPE = PneumaticsModuleType.CTREPCM;
```

Comment:
I believe there will be a single pneumatics controller, which will be a CTREPCM, it's not really a problem, but you only really need to be declaring that once

Reply:
yessir thanks https://github.com/flamingchickens1540/bunnybots-elmo-2022/blob/3b905b8bd5fc935b66e04e30b125a4211ac73cbb/src/main/java/org/team1540/elmo/Constants.java#L28